app/javascript/components/supplemental_application/sections/ParkingInformationInputs.js

```diff
@@ -9,23 +9,24 @@ const Yes = 'Yes'
 const No = 'No'
 
 const ParkingInformationInputs = ({ form: { change, getFieldState }, values: { lease }, disabled = false }) => {
-  const [parkingSpaceAssigned, setParkingSpaceAssigned] = useState(lease && lease['monthly_parking_rent'] ? Yes : No)
-  const selectParkingSpaceAssigned = (event) => {
-    const { target: { value } } = event
-    setParkingSpaceAssigned(value)
+  const onChangeHasParkingSpace = ({ target: { value } }) => {
     if (value === No) {
       change(monthlyRentFieldName, '')
     }
   }
+
   const monthlyRentVisited = getFieldState(monthlyRentFieldName)?.visited
+
+  const hasParkingSpace = lease?.bmr_parking_space_assigned === Yes
+
   return (
     <>
       <FormGrid.Row>
         <FormGrid.Item>
           <SelectField
             label='BMR Parking Space Assigned?'
-            selectValue={parkingSpaceAssigned}
```

Comment:
I think we probably don't need a selectValue at all actually. None of our other select fields have values set this way. They get it from react-final-form. 

If we remove it here then I think we can remove it from SelectField entirely? If we have a case where the value doesn't match what's displayed exactly we can set that when setting the available options.